### PR TITLE
CRABS

### PR DESCRIPTION
**Pull request by lbnesquik**
Add a crab crate to cargo. 25 points. Fully chromed. Lightweight and military grade. Simple as.

### CODE REVIEW

**Comment by lbnesquik** [comment]
[You thought i was original?](https://github.com/Citadel-Station-13/Citadel-Station-13-RP/pull/873)

File: code/datums/supplypacks/hydroponics.dm [hunk 1]
```diff
@@ -88,6 +88,12 @@
 	containername = "Chicken crate"
 	access = access_hydroponics
 
+//What a boon!
+/datum/supply_pack/hydro/crabs
+	name = "THREE (3) free crabs"
+	containertype = /mob/living/simple_mob/animal/passive/crab
```
Comment (Mechoid) [inline code/datums/supplypacks/hydroponics.dm:94]: This is using the mob as the container? And is also a single crab.

Comment (lbnesquik) [reply code/datums/supplypacks/hydroponics.dm:94]: Oversight. Will correct later.

Comment (Atermonera) [reply code/datums/supplypacks/hydroponics.dm:94]: @lbnesquik any plans to work on this?

**Pull request closed without merging**